### DISCUSSION TITLE
DeDRM not working

### DISCUSSION THREAD

**Discussion by Scheitelhengst** [apprenticeharper/DeDRM_tools#2361] [General]
Please help me.

ApplicationPaletteChange event ignored
Using calibre Qt style: True
calibre Fehlerdiagnoseprotokoll
calibre 7.19  embedded-python: True
Windows-10-10.0.22631-SP0 Windows ('64bit', 'WindowsPE')
('Windows', '10', '10.0.22631')
Python 3.11.5
Windows: ('10', '10.0.22631', 'SP0', 'Multiprocessor Free')
Interface language: de
EXE path: C:\Program Files\Calibre2\calibre-debug.exe
Successfully initialized third party plugins: DeDRM (10, 0, 9)
calibre 7.19  embedded-python: True
Windows-10-10.0.22631-SP0 Windows ('64bit', 'WindowsPE')
('Windows', '10', '10.0.22631')
Python 3.11.5
Windows: ('10', '10.0.22631', 'SP0', 'Multiprocessor Free')
Interface language: de
EXE path: C:\Program Files\Calibre2\calibre-debug.exe
Successfully initialized third party plugins: DeDRM (10, 0, 9)
QPA platform: windows
devicePixelRatio: 1.25
logicalDpi: 96.0 x 96.0
physicalDpi: 113.41395348837209 x 113.70777202072539
[0.00] Starting up...
[0.00] Showing splash screen...
[1.27] splash screen shown
[1.27] Initializing db...
[1.28] db initialized
[1.28] Constructing main UI...
[1.55] GUI main window shown
[1.62] main UI initialized...
[1.62] Hiding splash screen
[1.64] splash screen hidden
[1.64] Started up in 1.64 seconds with 1 books

**Comment by Scheitelhengst** [comment]
![Screenshot 2024-10-07 184241](https://github.com/user-attachments/assets/d65b0c3e-9346-4f30-9bf7-4719b33eb109)

**Reply by ElleKayEm** [reply]
Looks like you are trying to download a book that was already downloaded with a different account.  Until you get that straightened out, DeDRM won't be of any help.
